### PR TITLE
Read the list to do wum-live sync

### PR DESCRIPTION
**Pull request by Meruja**

### CODE REVIEW

File: wum-live-sync/loginScript.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/expect
+set update [lindex $argv 0];
+set carbonVersion [lindex $argv 1];
+
+# ssh to the instance
+spawn ssh -i /Users/meruja/Desktop/Auto-Patch-Signer-Needed-files/wum-live-sync/automationwumsynctesting.key ubuntu@192.168.114.55
```
Comment (IrushiL) [inline wum-live-sync/loginScript.sh]: Can we make this more generic? For instance let this value be set through a parameter. It is alright if there isn't a need to make it generic.

File: wum-live-sync/loginScript.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+set timeout 15
+
+# Set passphrase for the instance
+set myPassphrase "wso2carbon"
```
Comment (IrushiL) [inline wum-live-sync/loginScript.sh]: Is it alright to expose this value?

File: wum-live-sync/parallel-trigger-job.groovy [hunk 3]
```diff
@@ -107,17 +107,28 @@ def resultsAnalysis() {
...
 
     //path to script file for logging into the instance and start live syncing
-    def pathToScript = '/Users/parkavi/Documents/Parkavi/scripts/loginScript.sh'
-    def proc = pathToScript.execute()
-    proc.waitForOrKill(10000)
+    String carbonVersion = "wilkes"
+    //releasedUpdates =[5363,5546]
+    for (int i=0; i < releasedUpdates.size; i++) {
+        //full path to loginScript.sh
+        def cmd = ['/Users/meruja/gitRepo/forked/Final-APS/wso2-patch-signer/wum-live-sync/loginScript.sh',
```
Comment (IrushiL) [inline wum-live-sync/parallel-trigger-job.groovy]: This looks like a local path. Aren't we running this code inside some instance?

File: wum-live-sync/loginScript.sh (hunk 1 above)
Comment (Meruja) [reply wum-live-sync/loginScript.sh]: made the changes

File: wum-live-sync/loginScript.sh (hunk 2 above)
Comment (Meruja) [reply wum-live-sync/loginScript.sh]: It has been removed

File: wum-live-sync/parallel-trigger-job.groovy (hunk 3 above)
Comment (Meruja) [reply wum-live-sync/parallel-trigger-job.groovy]: Removed

**Pull request merged**